### PR TITLE
@jonahweissman's edits for clarity and stuff

### PR DESCRIPTION
**Pull request by jonahweissman**
Lots of changes, feel free to dispute them in a review.

### CODE REVIEW

File: engineering/1_hello_git.md [hunk 1]
```diff
@@ -3,46 +3,47 @@
 Git is an open source distributed version control system. Which means...
 
 * Open Source
-  * Git is free to use, alter, edit, resell, and mess around with however anyone sees fit.
+  * You are free to use, alter, edit, resell, and mess around with Git however you see fit.
 * Distributed
-  * It is very easy to collaborate with others using Git
+  * No single point of failure. If you are using Git to collaborate, everyone
```
Comment (Tim-Jackins) [inline engineering/1_hello_git.md]: Why have this be multi-line?

File: engineering/1_hello_git.md [hunk 2]
```diff
@@ -3,46 +3,47 @@
...
+    on your team has a full backup of the entire project.
 * Version Control
-  * Git tracks all the changes your make to all your files so you don't have too! This makes reverting easy and losing files hard.
```
Comment (Tim-Jackins) [inline engineering/1_hello_git.md]: Why have this be multi-line?

File: engineering/1_hello_git.md [hunk 3]
```diff
@@ -3,46 +3,47 @@
...
+  * Git remembers the history of all your files so you don't have to!
+    This makes reverting to previous versions easy and losing files hard.
 
-Git should probably already be installed on your raspberry pi. Type `git --version` in your terminal and if it returns a version number higher than 1.7.10, you're set!
```
Comment (Tim-Jackins) [inline engineering/1_hello_git.md]: Why have this be multi-line?

File: engineering/1_hello_git.md [hunk 4]
```diff
@@ -3,46 +3,47 @@
...
+Git should already be installed on your Raspberry Pi. Run the command
+`git --version` in your terminal and make sure it returns a version number
+higher than 1.7.10.
 
-Your probably filling out forms in all your other classes to let your teachers know who you are. So might as well do the same with Git?
```
Comment (Tim-Jackins) [inline engineering/1_hello_git.md]: Why have this be multi-line?

**Comment by Tim-Jackins** [comment]
I don't feel like long text needs to be split it up. I personally feel like it looks worse. Other than that I appreciate you cleaning up unused stuff in the repo and all your edits look grrrreat.

**Comment by jonahweissman** [comment]
Thank you for the kind words, Jack.

> I don't feel like long text needs to be split it up. I personally feel like it looks worse.

You are certainly entitled to your dislike of breaking text with new lines. However, the [commonmark spec indicates](https://spec.commonmark.org/0.28/#soft-line-breaks) that new lines like the ones I added will be rendered identically in the browser. You can confirm this by click the "Display the rich diff" button next to "view" for each file in the "Files changed" tab of this PR.

So, the only people who are affected are those who view the source (most likely in a terminal window). Why would wrapping long lines accomplish? Many of [this stack exchange answer's arguments apply](https://softwareengineering.stackexchange.com/a/1848), even though it's talking more about code than markdown. Some editors, such as `nano`, will extend long lines off the screen; you can scroll with arrow keys to read the whole line, but it's inconvenient.

For me, the main argument against these new lines is that it makes it inconvenient to edit the text because you have to move around where the new lines occur in the text. However, any editor worth its salt should be able to [handle this for you](https://vi.stackexchange.com/questions/11609/automatically-rewrap-lines-when-writing-markdown-in-vim).

Obviously, we should be consistent, and as maintainer, you make the final call. I just wanted you to make an informed decision.

**Comment by jonahweissman** [comment]
(I rebased so I could fix a typo)

**Comment by jonahweissman** [comment]
Those are very good reasons. I'm on board for doing them as one-liners

**Comment by Tim-Jackins** [comment]
I made 2 commits but since the branch is @jonahweissman:suggestions I thought it would be couth to have you look over them before I pulled.

**Comment by jonahweissman** [comment]
Looks good :+1:

**Pull request merged**